### ISSUE TITLE
Hey, I'm working on nottui :)

### ISSUE THREAD

**Issue by faldor20** [hedyhli/todomvc-tui#1]
Hi, I'm working on a fork of nottui, I saw it was on your list of TUI frameworks to use, please let me know if you want to use nottui, I have a lot of changes, a few bug fixes and a bunch of extra widgets and docs and such. 

This is a very cool project btw! 

Thanks :)

**Comment by hedyhli** [comment]
I'm very much interested in a fork if it has more docs! Is it ready yet? Including said extra docs? And out of curiosity, what's the reasoning behind the fork, or where can I read more about it?

I'm also open to accepting PRs if you'd like to implement the todo app yourself as part of examples or demo for your fork :)

**Comment by faldor20** [comment]
Not quite. I'm just writing a tutorial and I have a blog post digging in o the p icipals of layout and reactivity yout that should almost be ready.

Unfortunately won't be properly merged for quite some time as the maintaner of the project is busy with other work.

My fork is at  https://github.com/faldor20/lwd

Give me a few weeks and I'll have something nice for you :)

**Comment by hedyhli** [comment]
Ah, I see. I do appreciate your work on making the docs better, as well as (I just noticed!) Roc.

> Give me a few weeks and I'll have something nice for you :)

Take your time -- I'm quite caught up in the Zig implementation currently, and of course, waiting for minttea 0.0.3 to be published!

**Comment by faldor20** [comment]
I actually tried minttea for jj_tui at the start.  It was pretty rough sadly. I did use the latest version by just pinning it to the master branch on github. 
(if you're using opam that's `opam pin YOURURL`) 
but for some reason it just pinned my CPU at about 50% which was an already reported issue. Not really what I wanted for a light tui application 😅.  I'll follow how things go for you though :).
